### PR TITLE
People page layout modifications

### PR DESCRIPTION
**Pull request by Etiene**
### Motivation

- Update layout on team page and badger profile page
- Colours should be red instead of green
- The name and position should go below picture on team page
- Placeholder is now solid light grey

(Observation: because of branching issues, I had to rebase and remake this PR, which ended up on @dstodolny commits on the secondary image being carried along because it was merged to the old branch. This might result in circle CI not being able to build because it requires updates from badger brain)

### Test plan

- Check design on zeplin
- Check responsiveness on different browsers

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

File: site/pages/meet-our-team/style.css [hunk 1]
```diff
@@ -42,20 +42,26 @@
...
 
 a.category, a.category:hover, a.category:visited {
   color: badgerBlack;
 }
 
+a.category.active {
+  color: badgerWhite;
+}
+
+
```
Comment (dstodolny) [inline site/pages/meet-our-team/style.css]: \n\n :)

**Comment by samsmith1983** [comment]
Slight responsiveness issue on profile pages. Text doesn't stay the same width as image.

This is what happens currently:
screen shot 2017-02-13 at 17 03 28
(taken from here: https://www-staging.red-badger.com/59e2feb/about-us/people/sam-smith)

And it should look like this:

screen shot 2017-02-13 at 17 04 17
(taken from here: https://www-staging.red-badger.com/b229324/about-us/people/sam-smith)

**Comment by chris-brookes** [comment]
Bug - 
All desktop browsers:
Links on the surname and the arrow next to the name are not working. (but the first name and all area around works!). See highlighted area on the image attached.

screen shot 2017-02-13 at 17 31 54

**Comment by samsmith1983** [comment]
The above bugs are now fixed, and have not found any further issues 👍

**Comment by chris-brookes** [comment]
@Etiene - In IE11 small layout (<690px wide), if the person doesn't have a large amount of text in their details, they have a large gap between the picture and the name. All other browsers are fine.

**IE11.**
screen shot 2017-02-14 at 11 16 35


**Edge/FF/Chrome/safari**
screen shot 2017-02-14 at 11 13 36

**Comment by gregdorward** [comment]
Tested and passed the remaining issue, and retested the rest to make sure there's no regression - happy for this to be merged when we can (not sure if it needs to be parked seeing as it's a People Page change?)

**Pull request merged**